packages/ui/src/forum/components/threads/ThreadItem.tsx

```diff
@@ -147,7 +119,17 @@ export const ThreadItemWrapper = styled.a<{ $halfSize?: boolean }>`
   ${ThreadItemText} {
     -webkit-line-clamp: ${({ $halfSize }) => ($halfSize ? '3' : '14')};
   }
+`
 
+const StyledThreadItem = styled.a`
```

Comment:
Again please use `CardItem` instead of this one if possible to have only one component here